src/commands/wallet/balance.rs

```diff
@@ -3,39 +3,75 @@ use clap::Args as ClapArgs;
 use eyre::ContextCompat;
 use std::sync::Arc;
 use zksync_ethers_rs::{
-    contracts::erc20::ERC20,
+    contracts::{erc20::ERC20, l2_shared_bridge::get_l2_token_from_l1_address},
+    core::utils::{format_ether, format_units},
     providers::{Middleware, Provider},
     types::Address,
+    utils::L2_ETH_TOKEN_ADDRESS,
+    ZKMiddleware,
 };
 
 #[derive(ClapArgs)]
 pub(crate) struct Args {
     #[clap(long = "token")]
     pub token_address: Option<Address>,
+    #[clap(long = "tol2", required = false)]
```

Comment:
I suggest to name it `l2` to be consistent with the `l1` flag
```suggestion
    #[clap(long = "l2", required = false)]
```